applications/FSIApplication/tests/convergence_accelerator_spring_MPI_test.py

```diff
@@ -100,7 +102,7 @@ def InitializeNodalEpetraVector(self,model_part,rows_per_node):
     def setUp(self):
 
         # So far, the MPI convergence accelerator tests must be run with 2 processes
-        if (mpi.size != 2):
+        if (KratosMultiphysics.DataCommunicator.GetDefault().Size() != 2):
```

Comment:
```suggestion
        if KratosMultiphysics.ParallelEnvironment.GetDefaultSize() != 2:
```

Jordi recommended to use this one